examples/ui/ui.rs

```diff
@@ -128,11 +137,12 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                                         ..default()
                                     },
                                     ScrollingList::default(),
+                                    AccessibilityNode(NodeBuilder::new(Role::List)),
```

Comment:
These are fine for now, but needing to specify accessibility components manually kind of defeats the purpose of "accessibility by default", increases the burden on individual developers, and makes our UI examples more bloated / less approachable. I don't think its reasonable to expect your average game developer to remember to do this / do it correctly. Ideally our main "ui introduction example" has no explicit accessibility components.

Definitely cool with this as an initial step, but I think its very important to resolve this in the near future.